ui/components/app/connected-accounts-permissions/index.scss

```
@@ -3,7 +3,7 @@
 
   display: flex;
   flex-direction: column;
-  color: var(--Grey-500);
+  color: var(--color-icon-default);
```

If this is text it should be 
```suggestion
  color: var(--color-text-alternative);
```